scripts/setupTypeScript.js

```diff
@@ -17,7 +17,7 @@
 */
 
 import fs from "fs";
-import { dirname, path } from 'path';
+import { dirname }, path from 'path';
```

Comment:
Hey, are you sure this works? I just tried it in Glean.js and I have to do:

```js
import path from "path";
import { dirname } from "path";
```

Even in Javascript files.

Reply:
Well, [yes](https://github.com/mozilla-rally/rally-core-addon/blob/a3052afad8dbd443b40c7edd9ea0da4a64179bdc/tests/core-addon/integration/utils.js#L10)?

Reply:
Craziness. Why do I get an error...

Reply:
Ok, I have no documentation link, but I checked it out and it seems:
`import path, { dirname } from "path";` works,
`import { dirname }, path from "path";` doesn't.